modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/js/KaleoFormsViewRecordsManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {postForm} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {deleteDDLRecordURL},
+	portletNamespace,
+	...otherProps
+}) {
+	return {
+		...otherProps,
+		onActionButtonClick(event, {item}) {
+			if (item.data?.action === 'deleteRecords') {
+				if (
+					confirm(
+						Liferay.Language.get(
+							'are-you-sure-you-want-to-delete-this'
+						)
+					)
+				) {
+					const form = document.getElementById(
+						`${portletNamespace}searchContainerForm`
+					);
+
+					const searchContainer = document.getElementById(
+						otherProps.searchContainerId
+					);
+
+					if (form && searchContainer) {
```

Comment:
Should this be `(form || searchContainer)` instead?

Reply:
Both need to exist for `postForm` to make sense, so `&&` is correct.

Logic would be opposite if we had an early return, then it would be:
```js
if (!form || !searchContainer) {
    return;
}
```